playbook/app/pb_kits/playbook/pb_filter/Filter/CurrentFilters.jsx

```diff
@@ -20,6 +20,16 @@ const CurrentFilters = ({ dark, filters }: CurrentFiltersProps) => {
 
   return (
     <div className="maskContainer">
+      <If condition={isEmpty(filters)}>
+        <div className="filters">
+          <Title
+              paddingLeft="xs"
+              size={4}
+              tag="h4"
+              text="No Filter Selected"
+          />
```

Comment:
Can you use body light instead? I don't want people to get confused that they have active filters.

Reply:
I changed it from 'Title' to 'Body' instead. I'm not familiar with 'Body Light' as you mentioned... Not sure what that means.